test/kpod.bats

```diff
@@ -7,3 +7,33 @@ load helpers
 	echo "$output"
 	[ "$status" -eq 0 ]
 }
+
+@test "kpod pull from docker with tag" {
+	run ${KPOD_BINARY} pull debian:6.0.10
+	echo "$output"
+	[ "$status" -eq 0 ]
+}
```

Comment:
The rmi code has not been merged with cri-o yet - I can add that in once its merged.